### ISSUE TITLE
Accessibility problem with CommandBar flyout

### ISSUE THREAD

**Issue by jkgh2018** [microsoft/microsoft-ui-xaml#948]
**Describe the bug**
This bug was logged by the Accessibility team under MAS 1.3.1. Screenreader does not read the expanded/collapsed state for "More" app bar button

**Steps to reproduce the bug**
1. Launch an app which uses the command bar control and which has an overflow
2. Launch a screenreader (NVDA / Narrator)
3. Tab to the ellipsis button
4. Click ellipsis button. NVDA reads "more app bar button"
5. Click button again. NVDA reads "less app bar button"

**Expected behavior**
Screenreader should read the expanded/collapsed state for "More" app bar button.
User Impact - The state of the button (expanded/collapsed) is not conveyed to the screenreader users which may cause confusion to users about the content update present when button is expanded and what controls lie within the screen when button is not expanded.

**Screenshots**
![image](https://user-images.githubusercontent.com/42618470/60145342-9732bd80-977a-11e9-86b0-03c75200ccca.png)

**Version Info**

NuGet package version: 

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (xxxxx)              |    |
| May 2019 Update (18362)            |    |
| October 2018 Update (17763)        |    |
| April 2018 Update (17134)          |    |
| Fall Creators Update (16299)       |    |
| Creators Update (15063)            |    |
| Anniversary Update (14393)         |    |

| Device form factor | Saw the problem? |
| :-------------------- | :------------------- |
| Desktop                 |    |
| Mobile                   |    |
| Xbox                      |    |
| Surface Hub          |     |
| IoT                         |     |

**Additional context**

**Comment by YuliKl** [comment]
[MAS 1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}) requirement states:
*Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text.*

I have not seen any user reports about the state of the app bar being unclear. The More ("...") button changes its UIA name depending on app bar's overflow state. Implementing the expand/collapse pattern in this case would not provide further clarity to users. The More button is actually not the UI element that's expanding/collapsing, it's just controlling the visibility of app bar's overflow.

@jkgh2018, if you have an internal bug tracking this, please feel free to repath to OS\Core\DEP\UXP-User Experience Platform\Controls
